lib/parser/builders/default.rb

```diff
@@ -518,6 +518,15 @@ def pair_quoted(begin_t, parts, end_t, value)
       n(:pair, [ key, value ], pair_map)
     end
 
+    def pair_label(key_t)
+      key_l = loc(key_t)
+      value_l = key_l.adjust(end_pos: -1)
+
+      ident = n(:ident, [ value(key_t).to_sym ], Source::Map::Variable.new(value_l))
+      value = accessible(ident)
+      result = pair_keyword(key_t, value)
```

Comment:
```suggestion
      pair_keyword(key_t, value)
```

I think this assignment should be removed because it displays an unused variable warning under `RUBYOPT=-w` environment.